test/recipes/90-test_store.t

```diff
@@ -14,6 +14,32 @@ use OpenSSL::Test qw(:DEFAULT srctop_file srctop_dir bldtop_file data_file);
 my $test_name = "test_store";
 setup($test_name);
 
+# The MSYS2 run-time convert arguments that look like paths when executing
+# a program unless that application is linked with the MSYS run-time.  The
+# exact conversion rules are listed here:
+#
+#       http://www.mingw.org/wiki/Posix_path_conversion
+#
+# With the built in configurations (all having names starting with "mingw"),
```

Comment:
Isn't there dash missing? I mean as in "built-in configurations"?

Reply:
I think that english allows both forms, but may be wrong.  I'll change it.